### PR TITLE
Add a technology refresh page

### PR DESCRIPTION
**Pull request by CannonLock**
Article PR Template

Article previews can be found [here](https://chtc.github.io/article-preview/)

* [x] Linked the preview url `https://chtc.github.io/article-preview/<article-url>`
* [x] Looked at the link in https://socialsharepreview.com/ to verify socials
* [x] Requested reviews from the correct parties

### CODE REVIEW

**Comment by CannonLock** [comment]
https://chtc.github.io/article-preview/2022/08/31/Technology-Refresh.html
https://socialsharepreview.com/?url=https://chtc.github.io/article-preview/2022/08/31/Technology-Refresh.html

File: 2022-08-31-Technology-Refresh.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate
+    Education with funding from the Wisconsin Alumni Research Foundation, we have been able
+    to execute a major refresh of hardware.  This provided 207 new servers for our systems,
+    representing over 40,000 batch slots of computing capacity.
+
+card_src: "https://raw.githubusercontent.com/CHTC/Articles/main/images/PATh_Facility_Preview.jpeg"
+card_alt: Image of Servers
+---
+
+Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate Education with funding from the Wisconsin Alumni Research Foundation, we have been able to execute a major refresh of hardware.  This provided 207 new servers for our systems, representing over 40,000 batch slots of computing capacity. Most of this hardware arrived over the summer and we have started adding them to CHTC systems.
```
Comment (ChristinaLK) [inline 2022-08-31-Technology-Refresh.md:32]:
```suggestion
Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate Education with funding from the Wisconsin Alumni Research Foundation, CHTC has been able to execute a major refresh of hardware.  This provided 207 new servers for our systems, representing over 40,000 batch slots of computing capacity. Most of this hardware arrived over the summer and we have started adding them to CHTC systems.
```

File: 2022-08-31-Technology-Refresh.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+canonical_url: "https://chtc.cs.wisc.edu/Technology-Refresh.html"
+
+image:
+    path: "https://raw.githubusercontent.com/CHTC/Articles/main/images/PATh_Facility_Preview.jpeg"
+    alt: Image of Servers
+
+description: |
+    Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate 
+    Education with funding from the Wisconsin Alumni Research Foundation, we have been able
```
Comment (ChristinaLK) [inline 2022-08-31-Technology-Refresh.md:19]:
```suggestion
    Education with funding from the Wisconsin Alumni Research Foundation, CHTC has been able 
```

File: 2022-08-31-Technology-Refresh.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    alt: Image of Servers
+
+description: |
+    Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate 
+    Education with funding from the Wisconsin Alumni Research Foundation, we have been able 
+    to execute a major refresh of hardware.  This provided 207 new servers for our systems, 
+    representing over 40,000 batch slots of computing capacity.
+excerpt: |
+    Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate
+    Education with funding from the Wisconsin Alumni Research Foundation, we have been able
```
Comment (ChristinaLK) [inline 2022-08-31-Technology-Refresh.md:24]:
```suggestion
    Education with funding from the Wisconsin Alumni Research Foundation, CHTC has been able
```

File: 2022-08-31-Technology-Refresh.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Thanks to the generous support of the Office of the Vice Chancellor for Research and Graduate Education with funding from the Wisconsin Alumni Research Foundation, we have been able to execute a major refresh of hardware.  This provided 207 new servers for our systems, representing over 40,000 batch slots of computing capacity. Most of this hardware arrived over the summer and we have started adding them to CHTC systems.
+
+Continue reading to learn more about the types of servers we are adding and how to access them.
+
+### HTC System
+
+On the HTC system, we are adding 167 servers of new capacity, representing 36,352 job slots and 40 high-end GPU cards.
+
+The new servers will be running CentOS Linux 8 – see our previous email and website page about how to test your jobs and
```
Comment (ChristinaLK) [inline 2022-08-31-Technology-Refresh.md:40]:
```suggestion
The new servers will be running CentOS Linux 8 – CHTC users should see our website page about how to test your jobs and
```

**Review by ChristinaLK — APPROVED** [review]

**Pull request merged**